pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -358,9 +358,6 @@ func (m *kubeGenericRuntimeManager) getKubeletContainers(allContainers bool) ([]
 // getContainers lists containers by filter.
 func (m *kubeGenericRuntimeManager) getContainersHelper(filter *runtimeapi.ContainerFilter) ([]*runtimeapi.Container, error) {
 	resp, err := m.runtimeService.ListContainers(filter)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
Seems like we can `return m.runtimeService.ListContainers(filter)` directly ?

Reply:
+1

Reply:
already edited